### DISCUSSION TITLE
Can not display UI page when deploy as application

### DISCUSSION THREAD

**Discussion by nguyennam38** [ServiceStack/Discuss#73] [General]
I'm using Servicestack v6.10.0. When debug, I can open UI page and Admin page. When deploying to server, I add new application with name: NotificationServices
![s1](https://github.com/ServiceStack/Discuss/assets/12172654/2d4976b9-216c-441c-965a-e8e30b3149d5)

My service still run but have some error about js, css location path. js, css location path always fix to root path of website: http://localhost/css/bundle.css but correct path is: http://localhost/NotificationServices/css/bundle.css 
![s3](https://github.com/ServiceStack/Discuss/assets/12172654/da558333-4e00-4bbb-abbe-8fb4ac9fc08f)

I open UI page as link: http://localhost/NotificationServices/ui but web display the error: Failed to resolve module specifier "app", Failed to resolve module specifier "vue"
![s2](https://github.com/ServiceStack/Discuss/assets/12172654/4a5adfc3-9d38-48d5-9fca-4972107a37a7)

Can you support me to fix 2 errors above? 
Thank you.

**Comment by mythz** [comment]
Please provide a minimal stand-alone repro of the issue on GitHub which we can run to repro the issue
